### ISSUE TITLE
Problem witch highlighting preview in settings

### ISSUE THREAD

**Issue by sr-tream** [notepadqq/notepadqq#646]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/notepadqq/notepadqq/blob/master/CONTRIBUTING.md#question
```

**Current behavior**

Highlighting is not work in preview window on Settings->Appearance
https://i.imgur.com/dz2UyvI.png

**Expected behavior**

Highlighting is work as in early versions

**Minimal reproduction of the problem with instructions**

Open Settings->Appearance and choose any color scheme

**Please tell us about your environment:**

OS: Archlinux amd64
Qt: 5.10.1
DE: KDE Plasma 5.12.4
Notepadqq: 1.4 from repo

**Comment by JuBan1** [comment]
Already fixed in #640. Will hopefully land in the repos soon.

**Comment by JuBan1** [comment]
Already fixed in #640. Will hopefully land in the repos soon.
